README.md

```diff
@@ -104,6 +107,7 @@ PatchCamelyon [45a], [45b] | Lymph node |	H&E |	[data](https://patchcamelyon.gra
 PATHVQA [80] | Multiple | Multiple | [data](https://drive.google.com/drive/folders/1G2C2_FUCyYQKCkSeCRRiTTsLDvOAjFj5), [paper](https://arxiv.org/pdf/2003.10286.pdf), [github](https://github.com/UCSD-AI4H/PathVQA) | 32,799 open-ended questions from 4,998 images | image + question + answer | VQA | patch/image | | 2020
 Post-NAT-BRCA [74] | Breast | H&E | [data](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=52758117#52758117bcab02c187174a288dbcbf95d26179e8), [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6124665/) | 96 images from 54 patients | images + clinical info + annotation tumor cellularity and cell labels |  | wsi | 20x - Aperio | 2021
 Prostate Fused-MRI-Pathology [83] | Prostate | H&E | [data](https://www.cancerimagingarchive.net/collection/prostate-fused-mri-pathology/) | 114 images from 16 patients | images + tumor Annotations + mpMRI | | wsi | 20x - Aperio | 2016 
+RINGS | prostate | H&E | [data](https://data.mendeley.com/datasets/h8bdwrtnr5/1) | train: 1000 , test: 500 | images+mask |  gland segmentation and tumor segmentation | patch (1500x1500) | 40x | 2021
```

Comment:
Is this the associated paper: https://www.sciencedirect.com/science/article/abs/pii/S0933365721000695#:~:text=A%20new%20image%20segmentation%20method,traditional%20and%20deep%20learning%20techniques.
?
Also maybe add(18'851 glands?